server/test/users.test.js

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
```

Comment:
Why this space here?

Reply:
That is to separate the absolute imports from the relative imports

Reply:
Remove the space